cmd/rookflex/cmd/mount.go

```diff
@@ -244,6 +244,10 @@ func mountCephFS(client *rpc.Client, opts *flexvolume.AttachOptions) error {
 	)
 	if err != nil {
 		log(client, err.Error(), true)
+		log(client, "NOTE: Rook Flexvolume plugin uses 'mds_namespace' to refer to the ceph filesystem. "+
+			"This option is only available on Kernel v4.7 and higher. "+
+			"On older kernels, use the Kubernetes cephFS plugin to mount the filesystem. More details can be "+
+			"found in the Rook docs at https://rook.io/docs/rook/master/common-problems.html#filesystem-mounting", false)
```

Comment:
this change is dependent on #1189, right?  i'm wondering if the update to common-problems.md should be in this PR instead of #1189 that is supposed to be focused on updating the pending release notes.